Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -7777,6 +7777,45 @@ def string_to_context_key(string):
         raise Exception('The key is not a string: {}'.format(string))
 
 
+def response_to_context(reponse_obj, user_predefiend_keys):
```

Comment:
user_predefiend_keys is a positional argument and hence developers will need to do `response_to_context(my_dict, None)` in order to net get an error. 
let's add default value `None` to it
```suggestion
def response_to_context(reponse_obj, user_predefiend_keys=None):
```